projects/ngrx.io/content/marketing/docs.md

```diff
@@ -2,6 +2,44 @@
 
 NgRx is a framework for building reactive applications in Angular. NgRx provides state management, isolation of side effects, entity collection management, router bindings, code generation, and developer tools that enhance developers experience when building many different types of applications.
 
+## Why NgRx?
+
+NgRx is built for creating maintainable explicit applications, by encouraging the use of actions in order to express state changes.
+
+If you're building an application that has a lot of user interaction, many different event sources that's where NgRx is going to really shine in your application.
+
+
+### Serializability
+
+By normalizing state changes and pass them through observables NgRx provides serializability and good control of the application. This enables to save and serialize states to the `localStorage` or any other external storage.
+In addition, it also allows to inspect, download, upload, and dispatch actions, as well as time, travel all from the [Store Devtools](guide/store-devtools).
+
+
+### Type Safety
+
+NgRx is completely written in TypeScript.
+In NgRx, `reducers`, `selectors`, and `actions` are written in a specific way  get as much of type safety out of the TypeScript compiler as possible.
+
+
+### Encapsulation
+
+Using Ngrx [Effects](guide/effects) and [Store](guide/store),  any interaction with external resources side effects, like network requests, web socket and any business logic can be isolated from the UI. This isolation allows for more pure and simple components, and keep the single responsibility principle.
+
+### Testable
+
+Since NgRx is mostly based on pure factions like selectors and reducers, plus the ability to isolate side effects from the UI, testing becomes very straightforward.
```

Comment:
Did you mean `functions`?

```suggestion
Since NgRx is mostly based on pure functions like selectors and reducers, plus the ability to isolate side effects from the UI, testing becomes very straightforward.
```